api-report/container-loader.api.md

```diff
@@ -57,7 +57,7 @@ export interface IContainerLoadOptions {
     clientDetailsOverride?: IClientDetails;
     loadMode?: IContainerLoadMode;
     // (undocumented)
-    resolvedUrl: IFluidResolvedUrl;
+    resolvedUrl: IResolvedUrl;
```

Comment:
this is deprecated in main, and will be removed from public exports in branch in an upcoming PR